### ISSUE TITLE
Trap focus inside modal/dialog

### ISSUE THREAD

**Issue by jrobind** [jrobind/dev-jot#8]
When the lesson view modal appears there is no focus trap which is bad for assistive tech users. Currently, there is no way for a user to tab onto the close button and screen readers will also be unaware that a modal has appeared on screen.

![Screen Shot 2020-12-03 at 23 09 18](https://user-images.githubusercontent.com/30320536/101173476-80b0b700-363a-11eb-8cfa-f0e0238ed65d.png)

I suggest we follow the W3 spec: https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog
